src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -950,21 +950,27 @@ namespace winrt::TerminalApp::implementation
     void CommandPalette::SetActionMap(const Microsoft::Terminal::Settings::Model::IActionMapView& actionMap)
     {
         _actionMap = actionMap;
+        _setCommands();
     }
 
-    void CommandPalette::SetCommands(const Collections::IVector<Command>& actions)
```

Comment:
`SetCommands` was only ever being called in conjunction with `SetActionMap`, so I figured it'd make more sense for this to just be a private helper that `SetActionMap` calls